lib/karafka/templates/karafka.rb.example

```diff
@@ -18,7 +18,7 @@ class KarafkaApp < Karafka::App
     config.kafka.seed_brokers = %w( 127.0.0.1:9092 )
     config.client_id = 'example_app'
     config.backend = :inline
-    config.batch_consuming = true
+    config.batch_fetching = true
```

Comment:
This is by default set to `true`, maybe we can remove it?

Reply:
No because it indicates that such an option exists, so people starting with karafka are aware of it (and it is one of more important options)